frontend/src/components/ui/ActionModal/ActionModal.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './actionModal.css';
+import { Button } from '../Button';
+import { useEffect } from 'react';
+const ActionModal = ({
+  title,
+  subTitle,
+  content,
```

Comment:
please put `[]` as default, because currently logout modal cause crash, because of lack of content prop

Reply:
Also can you add more padding to inner content, in case content is not provided, only subTitle, so it looks more like logout modal in Figma. When content is present - paddings are OK now 

Thanks!

Reply:
to much padding between content's elements 
![image](https://github.com/user-attachments/assets/cbaaf021-2ac3-47b6-8f87-7e9501aa741b)